.github/workflows/update_file.yaml

```diff
@@ -0,0 +1,38 @@
+name: Update files
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '43 1 * * 1'
+
+jobs:
+
+  pull-from-sunbeam:
+    name: Pull files from sunbeam
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout local repository
+        uses: actions/checkout@v4
+
+      - name: set git config
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          git config --global user.email "${GITHUB_ACTOR_ID}+${GITHUB_ACTOR}@users.noreply.github.com"
+          git config --global user.name "$(gh api /users/${GITHUB_ACTOR} | jq .name -r)"
+          git config -l
+          
+      - name: Check pull from sunbeam
+        run: make sync-dashboards-alert-rules
+
+      - name: Create a PR for local changes
+        uses: peter-evans/create-pull-request@v6
+        id: cpr
+        with:
+          commit-message: "chore: update dashboards and alert rules"
+          title: "chore: update dashboards and alert rules"
+          body: |
+            Automated action to fetch the latest grafana dashboards and alert rules from Sunbeam. The branch of this PR 
+            will be overwritten during the next check. Unless you really know what you're doing, you 
+            most likely don't want to push any commits to this branch.
+          branch: chore/auto-libs
+          delete-branch: true
```

Comment:
Please configure your editor to auto-include the final trailing newline in files.  It's a good convention to follow, and most editors should do it by default.